tests/wasmfs/wasmfs_opfs.c

```diff
@@ -118,6 +118,11 @@ int main(int argc, char* argv[]) {
   assert(err == 0);
   emscripten_console_log("closed file");
 
+  err = stat("/opfs/working/foo.txt", &stat_buf);
+  assert(err == 0);
+  assert(stat_buf.st_size == 1);
+  emscripten_console_log("statted");
+
```

Comment:
In general it seems odd to do all this seemingly generic filesystem testing in an single opfs-specific test.  Don't we have good test coverage all the basic system call in existing tests?  Shouldn't we have some way to run the existing tests on different filesystem backend and writing every more tests for the same syscalls?

Reply:
Yes, we have a few mechanisms for running tests on multiple backends. This is our first persistent backend, though, so it has additional testing requirements. I don't know if we have other tests that would work as well as this one, especially since the error handling on the OPFS backend is not solid yet. We were planning to reuse this same test file for future persistent WasmFS backends, at least.

Reply:
But the legacy IDBFS and nodeFS and nodeRAWFS are all persistent too..  we test all the syscalls with those backends don't we?  

I'm worried about have so many different places were we test the same thing.  I think it would be good to spend some time auditing our different filesystem tests and coming up with a stratagy to simplify/consolidate (without loosing coverage).

Reply:
Yeah, that would be a good idea